### PR TITLE
replace ambiguous parameters regarding prune

### PR DESCRIPTION
**Pull request by monopole**
does this make more sense than true/false?

### CODE REVIEW

File: k8sdeps/transformer/inventory/inventory.go [hunk 1]
```diff
@@ -16,24 +16,35 @@ import (
...
 )
 
+type GarbagePolicy int
+
+const (
+	GarbageUnknown GarbagePolicy = iota
+	GarbageIgnore
+	GarbageCollect
+)
+
```
Comment (Liujingfang1) [inline k8sdeps/transformer/inventory/inventory.go]: You can define `GarbagePolicy` in `sigs.k8s.io/kustomize/pkg/types`. Then it won't introduce dependency on k8sdeps.

Comment (monopole) [reply k8sdeps/transformer/inventory/inventory.go]: good catch.  done!

**Comment by monopole** [comment]
thanks, done.

**Pull request merged**